AudioContextInterruptedState/explainer.md

```diff
@@ -0,0 +1,212 @@
+
+Given the snippet below, where the `AudioSession` type is set to `"auto"`, we would have the following behavior:
+1. Page is loaded and the AudioContext is `"suspended"`.
+2. After clicking on the "Play Audio" button, the `AudioContext` state will be `"running"`.
+3. The screen gets locked. At this moment, based on the the Audio Session API type, the UA should interrupt the `AudioContext`, and the context's state should now be `"interrupted"`.
+4. The screen gets unlocked. The UA should lift the interruption and the `AudioContext` should transition to the `"running"` state.
+
+```js
+<!DOCTYPE html>
+<html>
+  <body>
+    <button id="play-audio-btn">Play audio</button>
+    <script>
+      const audio_context = new AudioContext();
+      const oscillator = audio_context.createOscillator();
+      oscillator.connect(audio_context.destination);
+
+      const play_audio_btn = document.getElementById("play-audio-btn");
+
+      audio_context.addEventListener("statechange", () => {
+        console.log(audio_context.state);
+      });
+
+      play_audio_btn.addEventListener("click", () => {
+        oscillator.start();
+      });
+
+      window.addEventListener("load", () => {
+        navigator.audioSession.type = "auto";
+        console.log(audio_context.state);
+      });
+    </script> 
+  </body>
+</html>
+```
+
+## Web compatibility risks
+
+When `AudioContext.resume()` is called for an `AudioContext` in the `"closed"` state, the returned promise is rejected. With this proposal, the same behavior will happen when `AudioContext.resume()` is called while the `AudioContext` interrupted (could happen either during `"suspended"` or `"interrupted"`). In this case, a web page that is not aware of the existence of the "`interrupted`" state might think that the `AudioContext` has been closed.
+
+## Privacy considerations
+
+Moving the `AudioContexts` to the `"interrupted"` state might giveaway some information about the user's behavior - for example, when the user started a voice call. However, since "interrupting" an `AudioContext` could be associated to several types of interruptions, exposing this new state shouldn't be a major concern. Moreover, to mitigate privacy concerns, this proposal forbids automatically transitioning from `"suspended"` to `"interrupted"` to avoid unnecessarily exposing interruptions. This state transition will only happen if [`AudioContext.resume()`](https://webaudio.github.io/web-audio-api/#dom-audiocontext-resume) is called.

```

Comment:
nit: " started a voice call" --> " started a voice call or locked the screen"

Might as well call out the screen example from above.

Reply:
nit: "associated to" --> "associated with"